webknossos/webknossos/_nml/parameters.py

```diff
@@ -25,17 +26,38 @@ class Parameters(NamedTuple):
     taskBoundingBox: Optional[BoundingBox] = None
     userBoundingBoxes: Optional[List[BoundingBox]] = None
 
+    def _get_max_bbox_id(self) -> int:
```

Comment:
Could you add a docstring which explains what the maximum bbox id is (e.g, that only integer ids are considered), as the semantics weren't immediately clear to me because the ids can be strings too apparently.